ci-operator/step-registry/windows/unit/wicd/windows-unit-wicd-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: windows-unit-wicd
+  from: src
+  commands: windows-unit-wicd-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 400Mi
+  timeout: 2h15m0s
```

Comment:
I expected this test to be much shorter than our other e2e jobs. Any reason for the long timeout?

Reply:
You should lower the `requests` for resources too.